packages/airnode-node/src/evm/fulfillments/api-calls.test.ts

```diff
@@ -477,6 +478,51 @@ describe('submitApiCall', () => {
       }
     );
 
+    test.each([gasPrice, gasPriceFallback])(
+      `submits a fail transaction with a trimmed errorMessage for errored requests - %#`,
+      async (gasTarget: GasTarget) => {
+        const txOpts = { gasLimit: 500_000, ...gasTarget, nonce: 5 };
+        const provider = new ethers.providers.JsonRpcProvider();
+        const longError = 'This very long error message should get trimmed'.repeat(10);
+        const trimmedError = longError.substring(0, MAXIMUM_ONCHAIN_ERROR_LENGTH - 3).concat('...');
+        failMock.mockResolvedValueOnce({ hash: '0xfailtransaction' });
+        const apiCall = fixtures.requests.buildApiCall({
+          errorMessage: longError,
+          status: RequestStatus.Errored,
+          nonce: 5,
+        });
+        const [logs, err, data] = await apiCalls.submitApiCall(createAirnodeRrpFake(), apiCall, {
```

Comment:
Add `\n` for clarity before and after this call.

We do not have a strict convention for this, but one of the generic conventions for tests is "Arange/act/assert" where each section is separated by a new line (optionally preceeded by comment, which I hate).

Anyway, separating the sections is nice to do as the whole test reads better. But ofc you can create more groups when necessary or comment some non trivial stuff.

Reply:
Makes sense, thanks.